.github/workflows/test_macos.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
-      - run: yarn install --frozen-lockfile --check-files || yarn install --frozen-lockfile --check-files
+      - run: yarn install --immutable
```

Comment:
We had purposely ran this twice to retry on network errors, which are really common on macOS for some reason. Does Yarn 2 automatically retry?

Reply:
Indeed it does